mobile/examples/question_answering/android/app/src/main/java/ai/onnxruntime/example/question_answering/MainActivity.kt

```diff
@@ -0,0 +1,99 @@
+package ai.onnxruntime.example.question_answering
+
+import ai.onnxruntime.*
+import ai.onnxruntime.extensions.OrtxPackage
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.util.Log
+import android.widget.Button
+import android.widget.TextView
+import android.widget.Toast
+import androidx.activity.*
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.coroutines.*
+import java.util.*
+
+
+class MainActivity : AppCompatActivity() {
+    private var ortEnv: OrtEnvironment = OrtEnvironment.getEnvironment()
+    private lateinit var ortSession: OrtSession
+    private val TitleText: TextView by lazy {findViewById(R.id.TitleView)}
+    private val ArticleText: TextView by lazy {findViewById(R.id.ArticleView)}
+    private val AnswerText: TextView by lazy {findViewById(R.id.AnswerView)}
+    private val QuestionText: TextView by lazy {findViewById(R.id.QuestionView)}
+    private val DoQAButton: Button by lazy {findViewById(R.id.PerformQaButton)}
+
+    @SuppressLint("UseCompatLoadingForDrawables")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        // from <The Little Prince>, chapter one, and the first paragraph
+        TitleText.setText("<The Little Prince>")
+        ArticleText.setText(
+            "we are introduced to the narrator, a pilot, and his ideas about grown-ups." +
+                    "Once when I was six years old I saw a magnificent picture in a book," +
+                    " called True Stories from Nature, about the primeval forest. " +
+                    "It was a picture of a boa constrictor in the act of swallowing an animal. " +
+                    "Here is a copy of the drawing." +
+                    "In the book it said: \"Boa constrictors swallow their prey whole, " +
+                    "without chewing it. After that they are not able to move," +
+                    " and they sleep through the six months that they need for digestion.\""
+        );
+        QuestionText.setHint("From which book did I see a magnificent picture?")
+
+        // Initialize Ort Session and register the onnxruntime extensions package that contains the custom operators.
+        // Note: These are used to decode the input image into the format the original model requires,
+        // and to encode the model output into png format
+        val sessionOptions: OrtSession.SessionOptions = OrtSession.SessionOptions()
+        sessionOptions.registerCustomOpLibrary(OrtxPackage.getLibraryPath())
```

Comment:
When the super resolution app is set up, yes we were using this approach to setup registerCustomOpLibrary, but for now maybe we can try to update to registering custom op using a specified function name i.e. `session_options.RegisterCustomOpsUsingFunction("RegisterCustomOps")`
(which is a suggested more unified way in all mobile platforms.)

sample usage:
https://github.com/microsoft/onnxruntime/blob/34175f0b7cd949b6b719633f68fb223fa129fee5/onnxruntime/test/custom_op_registration/test_registercustomops.cc#L45-L47

original pr for reference: https://github.com/microsoft/onnxruntime/pull/14177

hopefully it does not require any additional setup. we should update the SuperRes one as well.

Reply:
It only works with embeded ort-extensions, is that right?

Reply:
I think the call of using function name should work with the current setup. but I can try with the super resolution android one and see if anything unexpected happens.

Reply:
handle = handle ? handle : RTLD_DEFAULT; // onnxruntime/core/platform/posix/env.cc:561

So `dlsym` will try to find the symbol recursively by its depended library.

If we didn't build ort with ort-exts, it's somhow unlikey to find it, isn't it?